packages/math/useClamp/index.ts

```diff
@@ -24,7 +34,7 @@ export function useClamp(
  * @param max
  */
 export function useClamp(value: MaybeRefOrGetter<number>, min: MaybeRefOrGetter<number>, max: MaybeRefOrGetter<number>) {
-  if (typeof value === 'function' || isReadonly(value))
+  if (typeof value === 'function' || isReadonly(value) || isCustomRef(value))
```

Comment:
why we need this? I ran the `should work with customRef` test in main branch with vue2, it's work fine.